Update synchronization-synchronizationjob-provision-on-demand.md (#17382)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/synchronizationlinkedobjects.md

---
title: "synchronizationLinkedObjects resource type"

> description: "Represents any references to be provisioned during on-demand provisioning."

Reviewer: ```suggestion
description: "Represents the principals to be provisioned during on-demand provisioning."
```

Author: I would prefer to keep it as references. We don't generally use the term principal in our experience. We call these things references (members, managers, owners). Today we support members. In the future, we will also add managers and owners.

author: "ArvindHarinder1"
ms.localizationpriority: medium
ms.prod: "applications"
doc_type: resourcePageType
---

# synchronizationLinkedObjects resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> References that you would like to provision on-demand such as a member of a group.

Reviewer: (Forgive my ignorance about provisioning) Considering that even devices can be [group members](https://docs.microsoft.com/en-us/graph/api/group-post-members?view=graph-rest-1.0&tabs=http), should we explicitly call out the type of principals that can be provisioned? Is it user principals only?

Q2: Should we also call out the group types that are allowed?
```suggestion
The principals that you would like to provision on-demand such as specific members of a group.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |members|[synchronizationJobSubject](../resources/synchronization-synchronizationjobsubject.md) collection|Any managers of the user that you would like to provision.|

Reviewer: Hmm, "managers" here intentional?

Author: Copy paste error. I'll fix. Thanks for catching.
